download/download.go

```diff
@@ -510,3 +507,30 @@ func GetURLsFile(urlsFilePath string) (urlsList []string, err error) {
 
 	return urlsList, scanner.Err()
 }
+
+func AnonymizeFilepath(filePath string) string {
```

Comment:
Nice refactoring! If this function is supposed to be used by other modules as well, I think it is better to be placed in `helpers.go`. 
And would be even better to add a comment to explain that it is trying to remove email like userID from the filepath. Maybe add a unit text as well?

Reply:
Fixed in 3a00170